src/views/GDPR.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+
+export const GDPR: React.FC = () => {
+  return (
+    <div>
+      <h1>Rekisteri- ja tietosuojaseloste</h1>
+      <h4>
+        Tämä on Tampereen Urheilusukeltajat ry:n (Taursu) EU:n yleisen
+        tietosuoja-asetuksen (GDPR) mukainen rekisteri- ja tietosuojaseloste.
+        Laadittu 30.11.2022. Viimeisin muutos 18.04.2023.
+      </h4>
+      <ol>
+        <li>
+          <h6>Rekisterinpitäjä</h6>
+          <span>Tampereen Urheilusukeltajat ry</span>
+          <br />
+          <span>Hyllilänkatu 1 B talli 26</span>
+          <br />
+          <span>33730 Tampere</span>
+          <br />
+          <br />
+          <span>Yhdistysrekisteritunnus 111.943</span>
+          <br />
+          <span>Y-tunnus 0592819-0</span>
+          <br />
+          <a href="mailto:jasenasiat.taursu@gmail.com">
+            jasenasiat.taursu@gmail.com
+          </a>
+        </li>
+        <br />
+        <li>
+          <h6>Rekisteristä vastaava yhteyshenkilö</h6>
+          <span>Akseli Kolari</span>
+          <br />
+          <a href="mailto:gdpr@taytttopaikka.fi">gdpr@tayttopaikka.fi</a>
+        </li>
+        <br />
+        <li>
+          <h6>Rekisterin nimi</h6>
+          <span>Täyttöpaikka -sovelluksen käyttäjärekisteri</span>
+        </li>
+        <br />
+        <li>
+          <h6>Oikeusperuste ja henkilötietojen käsittelyn tarkoitus</h6>
+          <p>
+            Henkilötietojen käsittelyn tarkoitus on tallentaa yhdistyksen
+            sukelluspullojen täyttöpaikalla tapahtuneet kaasujen täytöt
+            Turvallisuus- ja kemikaaliviraston määräysten mukaisesti, sekä
+            myöhempää mahdollista laskutusta varten. Tietoja ei käytetä
+            automatisoituun päätöksentekoon tai profilointiin.
+          </p>
+        </li>
+        <br />
+        <li>
+          <h6>Oikeusperuste ja henkilötietojen käsittelyn tarkoitus</h6>
+          <p>
+            Henkilötietojen käsittelyn tarkoitus on tallentaa yhdistyksen
+            sukelluspullojen täyttöpaikalla tapahtuneet kaasujen täytöt
+            Turvallisuus- ja kemikaaliviraston määräysten mukaisesti, sekä
+            myöhempää mahdollista laskutusta varten. Tietoja ei käytetä
+            automatisoituun päätöksentekoon tai profilointiin.
+          </p>
+        </li>
```

Comment:
```suggestion
        <li>
          <h6>Oikeusperuste ja henkilötietojen käsittelyn tarkoitus</h6>
          <p>
            Henkilötietojen käsittelyn tarkoitus on tallentaa yhdistyksen
            sukelluspullojen täyttöpaikalla tapahtuneet kaasujen täytöt
            Turvallisuus- ja kemikaaliviraston määräysten mukaisesti, sekä
            myöhempää mahdollista laskutusta varten. Tietoja ei käytetä
            automatisoituun päätöksentekoon tai profilointiin.
          </p>
        </li>
```
XD sama kama kahdesti

Reply:
Hitto
:D